requirements.txt

```diff
@@ -0,0 +1,2 @@
+autopep8
```

Comment:
it is recommended to keep consistent styling across the team, hence the requirement